tutorial/README.md

```diff
@@ -20,7 +20,7 @@ It supposes you have access to an instance of Transition, with access to a routi
 
 [GTFS](https://gtfs.org/) is a open standard format used to distribution transit information. Most agencies in the world use this format, that can be used by various tools to show transit data to users.
 
-Many transit feeds can be easily found in the [Open Mobility data](https://transitfeeds.com/) website, but the most up to date versions are recorded in the [Mobility Database catalogs](https://database.mobilitydata.org/). The transit agencies website may also contain this information. For example, the Montreal Transportation Society, whose data has been used to write this tutorial, can be found on [their website](https://www.stm.info/en/about/developers).
+Many transit feeds can be easily found in the [Open Mobility data](https://transitfeeds.com/) website, but the most up to date versions are recorded in the [Mobility Database catalogs](https://database.mobilitydata.org/). The transit agencies website may also contain this information. For example, the Société de transport de Montréal, the Montreal transit society, whose data has been used to write this tutorial, can be found on [their website](https://www.stm.info/en/about/developers).
```

Comment:
I would prefer Montreal transit agency instead is society which sound weird in English